posthog/hogql/grammar/HogQLParser.py

```diff
@@ -0,0 +1,16415 @@
+# Generated from HogQLParser.g4 by ANTLR 4.11.1
+# encoding: utf-8
+import sys
+from io import StringIO
+
+from antlr4 import *
+
+if sys.version_info[1] > 5:
+    from typing import TextIO
+else:
+    from typing.io import TextIO
+
+
+def serializedATN():
+    return [
+        4,
+        1,
```

Comment:
The newlines make this file HUGE. Can we exclude the whole of `grammar/` from `black` formatting?